File: RukaLjubavi.Api/Contracts/Requests/OcjenaDonacijeInsertRequest.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace RukaLjubavi.Api.Contracts.Requests
{
    public class OcjenaDonacijeInsertRequest
    {
        public string Komentar { get; set; }
        [Range(1, 5)]
        public int? Povjerljivost { get; set; }
        [Range(1, 5)]
        public int? BrzinaDostavljanja { get; set; }
        [Range(1, 5)]
        public int? PostivanjeDogovora { get; set; }
        public int OcjenjivacTipKorisnika { get; set; }
        public int KorisnikId { get; set; }
        public int DonacijaId { get; set; }
    }
}

File: RukaLjubavi.Android/app/src/main/java/com/fit/ba/rukaljubavi/Requests/OcjenaDonacijeInsertRequest.kt
package com.fit.ba.rukaljubavi.Requests

import java.io.Serializable

class OcjenaDonacijeInsertRequest()
{
    var komentar: String? = null
    var povjerljivost: Int? = null
    var brzinaDostavljanja: Int? = null
    var postivanjeDogovora: Int? = null
    var ocjenjivacTipKorisnika: Int? = null
    var korisnikId: Int? = null
    var donacijaId: Int? = null
}